### PR TITLE
Migrate monitoring pages to PF4 form components

### PR DESCRIPTION
**Pull request by rhamilto**
Supports https://jira.coreos.com/browse/CONSOLE-1564

Screen Shot 2019-07-11 at 9 51 54 AM

### CODE REVIEW

**Comment by rhamilto** [comment]
cc: @dtaylor113, @alecmerdler as @spadgett said y'all were working in this area.

File: frontend/public/components/monitoring.tsx [hunk 1]
```diff
@@ -820,7 +820,7 @@ const toISODate = (dateStr: string): string => {
   return isNaN(timestamp) ? undefined : (new Date(timestamp)).toISOString();
 };
 
-const Text = props => <input {...props} type="text" className="form-control form-control--silence-text" />;
+const Text = props => <input {...props} type="text" className="pf-c-form-control pf-c-form-control--silence-text" />;
```
Comment (kyoto) [inline frontend/public/components/monitoring.tsx]: `pf-c-form-control--silence-text` can be removed

File: frontend/public/components/_monitoring.scss [hunk 2]
```diff
@@ -40,10 +40,6 @@ $monitoring-line-height: 18px;
   color: $color-pf-blue-400;
 }
 
-.form-control--silence-text {
```
Comment (kyoto) [inline frontend/public/components/_monitoring.scss]: I think we want to keep this rule, just renamed to `pf-c-form-control--silence-text`.

Comment (rhamilto) [reply frontend/public/components/_monitoring.scss]: Good catch, Andy.  I removed the rule, but forgot to remove the class from the jsx.  

We don't actually want the class as it causes a layout bug with the new PF4 input class.
Screen Shot 2019-07-12 at 8 52 44 AM
Dropping the class gives us a consistent layout with the text area below..
Screen Shot 2019-07-12 at 8 52 34 AM

**Comment by rhamilto** [comment]
@spadgett comment addressed.  Please retag.

File: frontend/public/components/_monitoring.scss (hunk 2 above)
Comment (kyoto) [reply frontend/public/components/_monitoring.scss]: 👍 Thanks for the explanation @rhamilto

This does now differ from the mocks though. WDYT @cshinn? Are you happy with the 100% Creator input width, or should we bring back the max width like below?

screenshot

**Pull request merged**